pkg/controllers/hpareplicassyncer/hpa_scale_ref_worker.go

```diff
@@ -0,0 +1,134 @@
+package hpareplicassyncer
+
+import (
+	"context"
+	"fmt"
+
+	autoscalingv2 "k8s.io/api/autoscaling/v2"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog/v2"
+
+	"github.com/karmada-io/karmada/pkg/util"
+	"github.com/karmada-io/karmada/pkg/util/helper"
+)
+
+type labelEventKind int
+
+const (
+	addLabelEvent labelEventKind = iota
+	deleteLabelEvent
+)
+
+type labelEvent struct {
+	kind labelEventKind
+	hpa  *autoscalingv2.HorizontalPodAutoscaler
+}
+
+func (r *HPAReplicasSyncer) ReconcileScaleRef(key util.QueueKey) (err error) {
```

Comment:
This method seems do not need to be exported.

Reply:
done